### ISSUE TITLE
Wanderer distress hails in English before you can understand them

### ISSUE THREAD

**Issue by guillochon** [endless-sky/endless-sky#1123]
Just noticed that Wanderer ships will still send out distress hails in plain English before you can understand their language. Would be nice to have their distress hails be incomprehensible at this point in the storyline.

**Comment by LocalGod79** [comment]
#951 keeps looking more attractive.

**Comment by endless-sky** [comment]
Fixed in e814753.

**Comment by endless-sky** [comment]
Closing this because it's fixed in 0.9.1, which is released now.
